Slim/Player/Protocols/HTTP.pm

```diff
@@ -696,8 +696,7 @@ sub getMetadataFor {
 	if ( my $currentTitle = Slim::Music::Info::getCurrentTitle( $client, $url ) ) {
 		my @dashes = $currentTitle =~ /( - )/g;
 		if ( scalar @dashes == 1 ) {
-			($artist, $title) = split / - /, $currentTitle;
-		}
+			($artist, $title) = $currentTitle =~ /(.+)[ ]+-[ ]+(.+)/s;
```

Comment:
If you really wanted to back-port this, then please do back-port what we have in LMS8.

Reply:
No that came back by mistake when i was adding the other proposed change